apps/docs/src/components/document/patterns/page-not-found.tsx

```diff
@@ -16,7 +16,7 @@ export function PageNotFound() {
         <Paragraph>
           If the web address is correct or you selected a link or button,
           contact the <Link href="#">service</Link> Helpline if you need to
-          speak to someone about your service.
+          speak to someone about your [service].
```

Comment:
Missed it in my last PR. Syncing it with the pattern from [from the govie frontend](https://ogcio.github.io/ogcio-ds/?path=/docs/templates-page-not-found--docs)